LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Why not a more common MIT? As far as I know they are [compatible](https://opensource.stackexchange.com/a/582) and MIT is more adopted in the npm universe.

Reply:
Great inspiration for this project was [STRV's code-quality-tools repo](https://github.com/strvcom/code-quality-tools). Their code is under this license which demands same license of forks or derived projects. I do not want to pretend that this project is not based on others projects or other developers work.

I have no problem with MIT but on the other hand I would like to give a credit to my resources.

Reply:
Got it, so it's a price for inspiration 🙂. OK then!